lib/txgh/empty_resource_contents.rb

```diff
@@ -0,0 +1,42 @@
+module Txgh
+  class EmptyResourceContents
+    attr_reader :tx_resource
+
+    def initialize(tx_resource)
+      @tx_resource = tx_resource
+    end
+
+    def phrases
+      []
+    end
+
+    def write_to(stream, language = nil)
+    end
+
+    def to_s(language = nil)
+      ''
+    end
+
+    def to_h
+      {}
+    end
+
+    def diff(other_contents)
+      other_contents
+    end
+
+    def diff_hash(other_contents)
+      DiffCalculator::INCLUDED_STATES.each_with_object({}) do |state, ret|
+        ret[state] = []
+      end
+    end
+
+    def merge(other_contents, diff_hash)
+      other_contents
+    end
+
+    def empty?
+      true
+    end
+  end
```

Comment:
I would suggest splatting-out all of the unused args here in case the underlying APIs change, e.g.

```ruby
def write_to(*)
  # no-op
end
```

...and make `no-ops` explicit rather than blank methods